include/openthread/dns_client.h

```diff
@@ -199,6 +199,7 @@ typedef void (*otDnsAddressCallback)(otError aError, const otDnsAddressResponse
  *
  * @retval OT_ERROR_NONE          Query sent successfully. @p aCallback will be invoked to report the status.
  * @retval OT_ERROR_NO_BUFS       Insufficient buffer to prepare and send query.
+ * @retval
```

Comment:
Empty `@retval`?

Reply:
Thanks. Forgot to update the error code here.
Updated in new push.